iscsi/targetd/cmd/start.go

```diff
@@ -30,55 +29,52 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 )
 
-var log = logrus.New()
+//var log = logrus.New()
 
 // start-controllerCmd represents the start-controller command
 var startcontrollerCmd = &cobra.Command{
 	Use:   "start",
 	Short: "Start an iscsi dynamic provisioner",
 	Long:  `Start an iscsi dynamic provisioner`,
 	Run: func(cmd *cobra.Command, args []string) {
-		initLog()
-		log.Debugln("start called")
+		glog.V(2).Infoln("start called")
 		var config *rest.Config
 		var err error
 		master := viper.GetString("master")
 		kubeconfig := viper.GetString("kubeconfig")
 		// creates the in-cluster config
-		log.Debugln("creating in cluster default kube client config")
+		glog.V(2).Infoln("creating in cluster default kube client config")
 		if master != "" || kubeconfig != "" {
 			config, err = clientcmd.BuildConfigFromFlags(master, kubeconfig)
 		} else {
 			config, err = rest.InClusterConfig()
 		}
 		if err != nil {
-			log.Fatalln(err)
+			glog.Fatalln(err)
 		}
-		log.WithFields(logrus.Fields{
-			"config-host": config.Host,
-		}).Debugln("kube client config created")
+		glog.V(2).Infoln("kube client config created:", config.Host)
 
 		// creates the clientset
-		log.Debugln("creating kube client set")
+		glog.V(2).Infoln("creating kube client set")
 		kubernetesClientSet, err := kubernetes.NewForConfig(config)
 		if err != nil {
-			log.Fatalln(err)
+			glog.Fatalln(err)
 		}
-		log.Debugln("kube client set created")
+		glog.V(2).Infoln("kube client set created")
 
 		// The controller needs to know what the server version is because out-of-tree
 		// provisioners aren't officially supported until 1.5
 		serverVersion, err := kubernetesClientSet.Discovery().ServerVersion()
 		if err != nil {
-			log.Fatalf("Error getting server version: %v", err)
+			glog.Fatalf("Error getting server version: %v", err)
 		}
 
 		url := fmt.Sprintf("%s://%s:%s@%s:%d/targetrpc", viper.GetString("targetd-scheme"), viper.GetString("targetd-username"), viper.GetString("targetd-password"), viper.GetString("targetd-address"), viper.GetInt("targetd-port"))
 
-		log.Debugln("targed URL", url)
+		glog.V(2).Infoln("targed URL:", url)
 
 		iscsiProvisioner := provisioner.NewiscsiProvisioner(url)
-		log.Debugln("iscsi provisioner created")
+		glog.Infoln("iscsi provisioner created")
```

Comment:
You mix `glog.Infoln` and `glog.V(2).Infoln `. Can you make it consistent?

Reply:
to my understanding they are not the same thing .
the v(2) statements will be printed only when the flag -v>=2 is passed, the other statements will be always printed.
This is intentional in my code, basically the v(2) statements are the equivalent of debug level statements(for which there does not seem to exist a specific method in glog).